### DISCUSSION TITLE
Packard Bell PB450

### DISCUSSION THREAD

**Discussion by JasonShearer** [86Box/86Box#3419] [Ideas]
Not sure if this is the right way to go about this, but would it be possible to add support for the Packard Bell PB450?

The BIOS images can be found here (looking to emulate 1.1A PNP in particular, but support for all versions would be ideal): [https://theretroweb.com/motherboards/s/packard-bell-pb450#downloads](url)

More specs here: [http://www.uktsupport.co.uk/pb/mb/450.htm](url)

**Comment by OBattler** [comment]
This is already on the list of machines to be added.
